data/assets/util/webgui.asset

```diff
@@ -38,12 +38,12 @@ asset.onInitialize(function ()
     openspace.setPropertyValueSingle("Modules.WebGui.ServerProcessEnabled", enabled)
 
     if guiCustomization.webguiDevelopmentMode then
-        -- Route CEF to the deveopment version of the GUI.
+        -- Route CEF to the development version of the GUI.
         -- This must be manually served using `npm start`
         -- in the OpenSpace-WebGuiFrontend repository.
         openspace.setPropertyValueSingle(
             "Modules.CefWebGui.GuiUrl",
-            "http://127.0.0.1:4690/frontend/#/onscreen"
+            "http://127.0.0.1:4690/frontend/#/ontouch"
```

Comment:
We could possibly move this string to assets/customization/gui.asset, or perhaps just set a flag there and check it here.